dependencies/deployment/maven/deploy.sh

```diff
@@ -23,8 +23,8 @@ ARTIFACT="$ARTIFACT"
 MAVEN_PASSWORD="$MAVEN_PASSWORD"
 MAVEN_URL="$MAVEN_URL"
 MAVEN_USERNAME="$MAVEN_USERNAME"
-PARENT="$PARENT"
+COORDINATES="$COORDINATES"
```

Comment:
The value is a single maven coordinate like `ai.grakn:core.server:1.5.0`, right?  If yes we should call it `$COORDINATE` (singular) rather than `$COORDINATES` (plural)

Reply:
Actually, dismiss this comment.

We asked @jmsfltchr our english expert and we should use the plural form even when referring to just one since even for a single one it consists of a triplet (`ai.grakn`, `core.server`, `1.5.0`).